crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -432,6 +432,21 @@ input EventFilter {
 	eventType: String
 }
 
+type ExecutionResult {
+	"""
+	The effects field captures the results to the chain of executing this transaction
+	"""
+	effects: TransactionBlockEffects
+	"""
+	The errors field captures any errors that occurred during execution
+	"""
+	errors: [String!]!
+	"""
+	The digest field captures the digest of the transaction block
+	"""
+	digest: String!
```

Comment:
> This should be accessible via the effects, so we didn't have it in the draft schema, was there a rationale for exposing it here? I suppose there are some cases where we don't get effects because the transaction failed to sign, and we would want to communicate that.

Indeed. Also I wasn't sure about the fate of the effects field so having something one can use to refer to the tx was nice.

> Similar question for errors actually -- in the draft this is a single String, to match how the errors show up in SuiExecutionStatus -- when do we leverage the fact that it's a list? (i.e. when do we get multiple errors)?

No strong reason here. The plurality of the name "errors" seemed like it should be a list to me. Okay changing.

> Also can we make it nullable, so that if there are no errors, we can indicate that by returning None?

Sure